supporting_content/attributes/datetime_requirements.md

```diff
@@ -0,0 +1,37 @@
+# Datetime Requirements
+
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider  | Data set                 | Column                                               |
+|:----------|:-------------------------|:-----------------------------------------------------|
+| AWS       | CUR                      | lineItem/UsageStartDate, lineItem/UsageEndDate, etc. |
+| GCP       | Big Query Billing Export | usage_start_time, usage_end_time, etc.               |
+| Microsoft | Cost details             | date, etc.                                           |
+
+### Documentation
+
+* AWS: ?
```

Comment:
Do we want to leave this like this? Does it imply that the links are not ready yet OR that they will be added in the future when they are created?

Reply:
I believe these to be the docs that detail the columns that are datetime but they aren't very clear on the format they use or will use.  

* AWS: https://docs.aws.amazon.com/cur/latest/userguide/Lineitem-columns.html#Lineitem-details-U
* GCP: https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables/standard-usage
* Microsoft: https://learn.microsoft.com/en-us/azure/cost-management-billing/automate/understand-usage-details-fields

Reply:
@ijurica  thoughts?

Reply:
the idea was to replace the question marks prior to merge to working draft :)

Reply:
added references to providers documentation